### DISCUSSION TITLE
Environment HDR map shared assets

### DISCUSSION THREAD

**Discussion by Lemovision** [google/model-viewer#2913] [General]
Hi I'm trying to replicate a similar lighting as model-viewer "neutral" inside Unity
Since I have a unity webgl app that generates textures dynamically which loads into model-viewer
I found some old HDR files in this repository, for previous versions of the "default" lighting I believe (lightroom_14b.hdr)

https://github.com/google/model-viewer/tree/master/packages/shared-assets/environments
https://github.com/google/model-viewer/issues/521

Is it possible to have updated HDR file equivalent to the current version, especially for "neutral" lighting?

**Comment by elalish** [comment]
Good idea; I already created that, just forgot to upload it: https://github.com/google/model-viewer/pull/2916

**Comment by Lemovision** [comment]
Works great, thanks!

Btw for other Unity users who might search for it
Here's a tool to convert to cubemap for use in Unity
https://matheowis.github.io/HDRI-to-CubeMap
